documentation.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Documentation Strategy
+description: 
+published: true
+date: 2023-04-10T00:00:00.000Z
+tags: 
+editor: markdown
+dateCreated: 2023-04-10T00:00:00.000Z
+---
+
+This page describes our various documentation sources and seeks to eliminate gray area between them.
+
+## Docs site
+
+- Source: https://github.com/centerofci/mathesar/tree/master/docs
+- Published: https://docs.mathesar.org/
+- Scope:
+    - **User docs** (we don't have this yet, but we wil eventually)
+    - **Administrator docs** (for people installing/upgrading Mathesar)
+- How do edit:
```

Comment:
do --> to (also applies to other sections)

Reply:
Fixed in 0985d36b5eb17ea0b0ceb6f53addef4a930abe4b